app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        elif datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("All friends should be vaccinated")
+        elif visitor["wearing_a_mask"] is False:
+            raise (NotWearingMaskError
+                   ("Visitors must wear a mask to enter the cafe."))

```

Comment:
The parentheses around the `NotWearingMaskError` instantiation are unnecessary. You can directly raise the exception without them: `raise NotWearingMaskError("Visitors must wear a mask to enter the cafe.")`.